docs/branch-strategy.md

```
@@ -40,16 +40,46 @@ docs/#30-API-문서-작성
+| main 머지 (push) | CD (빌드 → 배포)               |
+| dev 푸시         | CI만 동작, 배포 없음           |
+
 ## 워크플로우
 
-1. **이슈 생성** — GitHub에서 이슈 템플릿을 사용하여 이슈를 생성합니다.
-2. **브랜치 생성** — dev에서 이슈에 대응하는 브랜치를 생성합니다.
+### 일반 개발
+
+1. **이슈 생성** - GitHub에서 이슈 템플릿을 사용하여 이슈를 생성합니다.
+2. **브랜치 생성** - dev에서 이슈에 대응하는 브랜치를 생성합니다.
    ```bash
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)